app/code/core/Mage/Eav/Model/Entity/Collection/Abstract.php

```diff
@@ -159,10 +159,12 @@ protected function _prepareStaticFields()
      */
     protected function _initSelect()
     {
-        $this->getSelect()->from(['e' => $this->getEntity()->getEntityTable()]);
-        if ($this->getEntity()->getTypeId()) {
+        $this->getSelect()->from(array('e' => $this->getEntity()->getEntityTable()));
+
+        if ($this->getEntity()->getEntityTable() === Mage_Eav_Model_Entity::DEFAULT_ENTITY_TABLE && $this->getEntity()->getTypeId()) {
```

Comment:
Resolved conflict with 1.9.4.x, short array:

```suggestion
        $this->getSelect()->from(['e' => $this->getEntity()->getEntityTable()]);
 
        if ($this->getEntity()->getEntityTable() === Mage_Eav_Model_Entity::DEFAULT_ENTITY_TABLE && $this->getEntity()->getTypeId()) {
```